features/FEATURE_LWIP/lwip-interface/lwipopts.h

```diff
@@ -80,19 +80,25 @@
 #define DEFAULT_RAW_RECVMBOX_SIZE   8
 #define DEFAULT_ACCEPTMBOX_SIZE     8
 
+// Thread stack size for lwip tcpip thread
+#ifndef MBED_CONF_LWIP_TCPIP_THREAD_STACKSIZE
```

Comment:
shouldn't this conf values be used (they are ignored as I see it as TCPIP_THREAD_STACKSIZE used in the lwip not this new conf value) ?